.github/workflows/land-blocking.yml

```diff
@@ -175,3 +153,70 @@ jobs:
             if (should_fail) {
               throw "Land-blocking test failed";
             }
+  build-and-run-cluster-test-k8s:
+    name: Run cluster test on k8s
+    needs: build-and-run-cluster-test
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup env
+        run: |
+          echo "::set-env name=LIBRA_GIT_REV::$(git rev-parse --short=8 HEAD)"
+          echo "::set-env name=TEST_TAG::land_$(git rev-parse --short=8 HEAD)"
+      - name: Check kill switch
+        id: check_ks
+        run: |
+          if ${{ secrets.KILL_SWITCH_LAND_BLOCKING_TEST }} ; then
+            echo "::set-output name=should_run::false";
+          else
+            echo "::set-output name=should_run::true";
+          fi;
+      - name: Launch cluster test
+        if: steps.check_ks.outputs.should_run == 'true'
+        # NOTE Remember to update PR comment payload if cti cmd is updated.
+        run: |
+          export CTI_OUTPUT_LOG=$(mktemp)
+          ./scripts/cti \
+            --k8s \
+            --tag ${TEST_TAG} \
+            --report report-k8s.json \
+            --run bench \
+            --k8s-fullnodes-per-validator=0 \
+            --k8s-num-validators=100
+
+          echo "::set-env name=CTI_OUTPUT_LOG::$CTI_OUTPUT_LOG"
+          echo "report-k8s.json start"
+          cat report-k8s.json
+          echo "report-k8s.json end"
+      - name: Post test results on PR
```

Comment:
I think we can skip this step so that it won't spam the PR.

Reply:
So it does not actually post anything to the PR, it just validates if the TPS report met the required criteria

Reply:
Renamed the step